fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,6 +34,16 @@
 
 	</servlet-mapping>
 
+  <filter>
+    <filter-name>ETagFilter</filter-name>
+    <filter-class>org.springframework.web.filter.ShallowEtagHeaderFilter</filter-class>
```

Comment:
As noted in the java docs, the `ShallowEtagHeaderFilter` saves bandwidth, but the server still needs to calculate the checksum. 
Since the static content is... static, we could save performance as well by overwriting the method [ShallowEtagHeaderFilter.generateEtagHeaderValue](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/filter/ShallowEtagHeaderFilter.html#generateETagHeaderValue-java.io.InputStream-boolean-).